CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ## Upcoming Release
 
 ### Added
+- **RawRepresentable**:
+  - Conform to `Equatable` when `RawValue` is `Equatable`. [#804](https://github.com/SwifterSwift/SwifterSwift/pull/804) by [guykogus](https://github.com/guykogus)
```

Comment:
This was removed from this PR, so we should remove this entry from changelog :))